### PR TITLE
[#12048] Move accounts JSON for AutomatedSessionRemindersE2ETest

### PR DESCRIPTION
**Pull request by weiquu**
Part of #12048

### CODE REVIEW

File: src/e2e/resources/data/AutomatedSessionRemindersE2ETest_SqlEntities.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "accounts": {
+    "instructorWithEvals": {
+      "googleId": "tm.e2e.AutSesRem.instructor",
+      "name": "Test Ins for Aut Sessions Reminder",
+      "email": "AutSesRem.instructor@gmail.tmt",
+      "readNotifications": {}
+    }
```
Comment (EuniceSim142) [inline src/e2e/resources/data/AutomatedSessionRemindersE2ETest_SqlEntities.json:8]: Need to add id and remove readNotifications (according to Account entity):
```suggestion
    "instructorWithEvals": {
      "id": "00000000-0000-4000-8000-000000000001",
      "googleId": "tm.e2e.AutSesRem.instructor",
      "name": "Test Ins for Aut Sessions Reminder",
      "email": "AutSesRem.instructor@gmail.tmt",
    }
```

**Review by EuniceSim142 — CHANGES_REQUESTED** [review]
lgtm 👍 , other than 1 thing

**Review by EuniceSim142 — APPROVED** [review]

File: src/e2e/java/teammates/e2e/cases/AutomatedSessionRemindersE2ETest.java [hunk 2]
```diff
@@ -49,6 +49,8 @@ protected void prepareTestData() {
         // Published time for one feedback session already set to some time in the past.
 
         removeAndRestoreDataBundle(testData);
+
```
Comment (cedricongjh) [inline src/e2e/java/teammates/e2e/cases/AutomatedSessionRemindersE2ETest.java:52]: change line 29

**Review by cedricongjh — APPROVED** [review]
lgtm

**Pull request merged**